src/main/java/persistence/sql/domain/FieldType.java

```diff
@@ -0,0 +1,32 @@
+package persistence.sql.domain;
+
+public enum FieldType {
+    LONG(Long.class, "BIGINT"),
+    STRING(String.class, "VARCHAR"),
+    INTEGER(Integer.class, "INT");
```

Comment:
해당 커멘트는 개인적인 제안입니다.

```suggestion
    LONG(Long.class, "BIGINT"),
    STRING(String.class, "VARCHAR"),
    INTEGER(Integer.class, "INT"),
    ;
```

이렇게 작성하게 되면 나중에 타입이 추가됐을 때 conflict을 방지할 수 있는 이점이 있습니다.